Automattic/wp-calypso
# Subscriber management: Show pretty subscriber count

Related to https://github.com/Automattic/jetpack/issues/32373

## Proposed Changes

It shows big subscriber numbers more elegantly.

| Before | After |
| ------------- | ------------- |
| <img width="454" alt="Screenshot 2024-01-05 at 10 29 31" src="https://github.com/Automattic/wp-calypso/assets/4068554/c8c42aba-7f7a-4620-b5a1-e808251d7c79"> | <img width="454" alt="Screenshot 2024-01-05 at 10 29 11" src="https://github.com/Automattic/wp-calypso/assets/4068554/265ada3f-a35c-41ad-99b8-9d6a8c204b4f"> |

## Testing Instructions

Go to Users -> Subscribers and make sure the total subscriber number is displayed as expected.

## Pre-merge Checklist
